### PR TITLE
Spelling and grammer corrections in gsoc project md files

### PR DESCRIPTION
**Pull request by AdarshKumar712**
Few of the changes that I made that haven't been highlighted properly:
1. In science.md, "enhance" changed to "enhancing" in a highlighted paragraph.
                            "hyperrectangular" changed to "hyper rectangular" in second highlighted paragraph.
2. In tables.md, "engineering the" changed to "engineering of the" in last line of highlighted para.
3. In wasm.md, "julia" changed to "Julia" in first highlighted paragraph.
                        "ongoing to to" changed to "ongoing to"  in second line and "pre-requisite" changed to "prerequisite" in last line of second highlighted paragraph.

### CODE REVIEW

File: jsoc/gsoc/MLJ.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ References:
 
 MLJ is so far focused on tabular data. This project is to add support for time series data in a modular, composable way.
 
-Time series are everywhere in real-world applications and there has been an increase in interest in time series toolboxes recently (see e.g. [sktime](https://github.com/alan-turing-institute/sktime), [tslearn](https://github.com/rtavenar/tslearn), [tsml](https://github.com/uea-machine-learning/tsml/)).
+Time series are everywhere in real-world applications and there has been an increase in interest in time series tool boxes recently (see e.g. [sktime](https://github.com/alan-turing-institute/sktime), [tslearn](https://github.com/rtavenar/tslearn), [tsml](https://github.com/uea-machine-learning/tsml/)).
```
Comment (tlienart) [inline jsoc/gsoc/MLJ.md]: kindly change that for "frameworks" , "tool boxes" is weird.

File: jsoc/gsoc/diffeq.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ linear algebra, and the ability (or eagerness to learn) to write fast code.
 
 ## Discretizations of partial differential equations
 
-There are two ways to approach libraries for partial differential equations (PDEs): one can build "toolkits" which enable users to discretize any PDE but require knowledge
+There are two ways to approach libraries for partial differential equations (PDEs): one can build "toolkits" which enable users to discretize a PDE but require knowledge
```
Comment (tlienart) [inline jsoc/gsoc/diffeq.md]: why this change? any is fine here

**Comment by AdarshKumar712** [comment]
Updated! @tlienart. Thanks for your suggestions! :)

**Comment by tlienart** [comment]
Thanks!

**Comment by tlienart** [comment]
Ps: I just realised that your PR title has a massive typo in it which is hilarious, not sure if that was intended or not but either way it's pretty funny

**Comment by mforets** [comment]
@AdarshKumar712 thanks for fixing the typos in "Computational methods using zonotopes" :+1: 

> "hyperrectangular" changed to "hyper rectangular" 

[Hyperrectangular](https://en.wiktionary.org/wiki/hyperrectangular) which means having the form of a [hyperrectangle](https://en.wikipedia.org/wiki/Hyperrectangle), is correct though.

**Comment by AdarshKumar712** [comment]
Oh sorry! I thought that was a typo. Should this change be reverted @mforets?
In that case, if that's replaced with hyper-rectangular, would that be better?

**Comment by mforets** [comment]
No worries. Yes, "hyper-rectangular" is better than "hyper rectangular" 😄
